File: C/problema_menor_de_tres.c
#include <stdio.h>

#include <math.h>
int main()
{

  int a, b, c, menor;

  printf("Digite o primeiro valor: ");
  scanf("%d", &a);
  printf("Digite o segundo valor: ");
  scanf("%d", &b);
  printf("Digite o terceiro valor: ");
  scanf("%d", &c);

  if (a < b && a < c)
  {
    menor = a;
  }
  else if (b < c)
  {
    menor = b;
  }
  else
  {
    menor = c;
  }

  printf("O menor valor é: %d\n", menor);

  return 0;
}

File: Python/ProblemaMenorDeTres.py
primeiroValor: int; segundoValor: int; terceiroValor: int

primeiroValor = int(input("Primeiro valor: "))
segundoValor = int(input("Segundo valor: "))
terceiroValor = int(input("Terceiro valor: "))

if primeiroValor < segundoValor and primeiroValor < terceiroValor:
    print(f"MENOR = {primeiroValor}")
elif segundoValor < primeiroValor and segundoValor < terceiroValor:
    print(f"MENOR = {segundoValor}")
else: 
    print(f"MENOR = {terceiroValor}")